test/junit/scala/collection/mutable/OpenHashMapTest.scala

```diff
@@ -0,0 +1,42 @@
+package scala.collection.mutable
+
+import org.junit.Test
+import org.junit.Assert._
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+/** Tests for [[OpenHashMap]]. */
+@RunWith(classOf[JUnit4])
+class OpenHashMapTest {
+  /** Test that an [[OpenHashMap]] correctly maintains its internal `deleted` count. */
+  @Test
+  def maintainsDeletedCount {
+    import scala.reflect.runtime.{universe => ru}
```

Comment:
I moved it into the comment. The commented code is there so to switch to Scala reflection once SI-9306 is fixed. I wouldn't call the Java version "clean", since, unlike Scala, it depends on the name being expanded. I'm just writing Java reflection code for other (future) tests in this class; will convert those as well after SI-9306.